bridge/src/error.rs

```diff
@@ -19,10 +35,9 @@ error_chain! {
     }
```

Comment:
`cargo check` yields `lint unused_doc_comment has been renamed to unused_doc_comments`

Reply:
removed